include/flatbuffers/code_generators.h

```diff
@@ -74,9 +74,35 @@ class BaseGenerator {
     return true;
   }
 
-  std::string FullNamespace(const char *separator, const Namespace &ns) {
+  static void NamespacePrefixAsComponents(const std::string &prefix, std::vector<std::string> &namespaces)
```

Comment:
Google code style guide says non-const arguments like `namespaces` should be passed by `*`